ui/src/components/RequisitionDefinitionsLayout.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+    <div class="container">
+        <div class="btnAction">
+            <Button :icon="icon" :label="buttonAction[index]" @click="clickAction(index)"></Button>
+        </div>
+        <div class="reqDef">
+            <div class="dataTable" v-if="index === 0">
+                <dataTableDemo></dataTableDemo>
+            </div>
+            <div v-else>
+                <reqDefinitionFrom></reqDefinitionFrom>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+
+import { ref } from 'vue'
+import Button from './Common/Button.vue'
+import dataTableDemo from './Common/Demo/DataTableDemo.vue'
+import reqDefinitionFrom from './ReqDefinitionForm.vue'
```

Comment:
Let's keep the casing the same for all component imports. `DataTableDemo` etc.

Reply:
Done